### PR TITLE
Subparser addresses

### PR DESCRIPTION
**Pull request by hardyjosh**
Updating the Flare pubstrats with the Flare subparser and an Orderbook subparser that have authoring meta on the Flare metaboard.

### CODE REVIEW

File: src/learning/flare/flr-dca-cooldown.rain [hunk 1]
```diff
@@ -55,13 +54,11 @@ charts:
...
         value: 0.6
 ---
-#flare-sub-parser !The flare subparser address.
-
 #per-trade-amount !The amount that we'll buy or sell per trade.
 #cooldown-time !The minimum amount of seconds that must elapse between trades.
 
 #calculate-io
-using-words-from flare-sub-parser
+using-words-from 0xAfD94467d2eC43D9aD39f835BA758b61b2f41A0E 0x50fB1Be59721E7A2a3434dDd726d53780bfDe344
```
Comment (thedavidmeister) [inline src/learning/flare/flr-dca-cooldown.rain:61]: why did you remove this from being a binding?

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: src/learning/flare/flr-dca-cooldown.rain (hunk 1 above)
Comment (hardyjosh) [reply src/learning/flare/flr-dca-cooldown.rain:61]: Because the strats are specifically for flare anyway and I think this way it's a clearer example of how subparsers work

**Review by thedavidmeister — APPROVED** [review]
